toolkit/docs/formats/imageconfig.md

```diff
@@ -429,6 +429,15 @@ A sample KernelCommandLine enabling SELinux and booting in enforcing mode:
 },
 ```
 
+A sample KernelCommandLine enabling SELinux and overwriting the default 'selinux-policy' package name:
+
+``` json
+"KernelCommandLine": {
+    "SELinux": "enforcing",
+    "SELinuxPolicy": "my-selinux-policy"
```

Comment:
Perhaps `SELinuxPolicyPackage`? This just makes it explicit that it refers to an RPM package.

Reply:
I'd rather shorter name.